### DISCUSSION TITLE
Is there a way to update an LXC with the latest version of a script?

### DISCUSSION THREAD

**Discussion by bobloadmire** [tteck/Proxmox#2821] [Q&A]
For instance I see the Frigate script has been undergoing changes within the last month. My LXC was created before that, can I update it with the latest version of the script without breaking my LXC? Been having HW encode/decode issues, so I'm trying to see if anything helps.

**Comment by peter1rhodes** [comment]
I would also like to know how to update Frigate to the latest version when it's running in a tteck LXC

**Reply by tteck** [reply]
Me too.

**Reply by chadsell** [reply]
@tteck Usually your scripts will update the LXC if run inside the LXC instead of on the node. I've noticed that some don't, though.  My Prometheus LXC just returns a "no upgrade path available" error when I try.

Is this an error, or is it because there's no easy way to upgrade via script?

**Reply by InSanniti** [reply]
I'm also getting the same error with trying to update Prometheus with the update script:
https://github.com/tteck/Proxmox/blob/main/ct/prometheus.sh

"✗ There is currently no update path available."

Any workaround or fix for this @tteck  ?
